pkg/tracing/tracing.go

```diff
@@ -0,0 +1,104 @@
+package tracing
```

Comment:
Should this be moved to the `internal` directory?

Reply:
Done in d3a51f9e6e8a91eccf6c1dfb50b5ca0fba7a2f9e